manifests/moco-nodes.pp

```diff
@@ -18,6 +18,7 @@
     $slave_trustlevel = 'try'
     include fw::profiles::osx_taskcluster_worker
     include toplevel::worker::releng::generic_worker::test::gpu
+    include users::roller  # roller needs ssh to reboot macs
```

Comment:
Copied from mozreview:
> I think we should be doing ssh on the linux taskcluster host in addition to the osx hosts.  A ssh->reboot is a much more graceful and cleaner shutdown.  If that fails, roller should fall back to issuing IPMI shutdown.  This is also the model slaveapi uses.
I'm assuming the IPMI soft-reboot doesn't signal the os to reboot, if it does, disregard.  The goal is a graceful shutdown and then falling back to a hard reboot.
5/2/2018 [:dividehex]

Reply:
Copied from mozreview:
> The ipmi soft-reboot issues an ACPI shutdown and so I think it is gracefully caught by the OS. But let's add the roller user to the linux boxes also; we'll need ssh to the linux machines for creating loaners anyway.
[:dhouse]